Detectors/TPC/reconstruction/include/TPCReconstruction/TrackTPC.h

```diff
@@ -130,17 +129,19 @@ TrackTPC::TrackTPC(const std::array<float,3> &xyz,const std::array<float,3> &pxp
 {}
 
 inline
-void TrackTPC::addCluster(Cluster &c)
+void TrackTPC::addCluster(Cluster const &c)
```

Comment:
Don't we usually write `const Cluster&` instead of Cluster const &`

Reply:
true (my old habit)